sdk/src/main/java/com/horizen/account/api/rpc/types/EthereumBlock.java

```diff
@@ -0,0 +1,157 @@
+package com.horizen.account.api.rpc.types;
+
+import com.fasterxml.jackson.annotation.JsonView;
+import com.horizen.account.block.AccountBlock;
+import com.horizen.serialization.Views;
+import org.web3j.utils.Numeric;
+
+import java.util.List;
+
+@JsonView(Views.Default.class)
+public class EthereumBlock {
+    private String number;
+    private String hash;
+    private String parentHash;
+    private String nonce;
+    private String sha3Uncles;
+    private String logsBloom;
+    private String transactionsRoot;
+    private String stateRoot;
+    private String receiptsRoot;
+    private String author;
+    private String miner;
+    private String mixHash;
+    private String difficulty;
+    private String totalDifficulty;
+    private String extraData;
+    private String size;
+    private String gasLimit;
+    private String gasUsed;
+    private String timestamp;
+    private List<String> transactions;
+    private List<EthereumTransactionView> transactionViews;
+    private List<String> uncles;
+    private List<String> sealFields;
+    private String baseFeePerGas;
+
+    public EthereumBlock() {
+    }
+
+    public EthereumBlock(String number, String hash, List<String> transactions, List<EthereumTransactionView> transactionViews, Boolean hydrated, AccountBlock block) {
+        this.number = number;
+        this.hash = hash;
+        this.parentHash = "0x0";
+        this.nonce = "0x0"; // no nonce
+        this.sha3Uncles = "0x0"; // no uncles
+        this.logsBloom = "0x0"; // not included in block now
+        this.transactionsRoot = "0x0";
+        this.stateRoot = Numeric.toHexString(block.header().stateRoot());
+        this.receiptsRoot = Numeric.toHexString(block.header().receiptsRoot());
+        this.miner = "0x0";
+        this.mixHash = "0x0";
+        this.extraData = "0x0";
+        this.size = String.valueOf(block.header().bytes().length);
```

Comment:
I think it must be the size of the whole block.

Reply:
Where do we get it?